cmd/terramate/cli/cli.go

```diff
@@ -612,6 +614,10 @@ func (c *cli) checkLocalDefaultIsUpdated(g *git.Git) error {
 	return nil
 }
 
+func (c *cli) showdir(dir string) (string, bool) {
+	return prj.ShowDir(c.root(), c.wd(), dir)
+}
```

Comment:
hmm :thinking: ...starting to think that it is some sort of canonical dir representation, but not sure yet =P